resources/datadog-integrations/project.rb.erb

```diff
@@ -48,6 +50,15 @@ runtime_dependency 'datadog-agent'
 
 extends_packages('datadog-agent', 'datadog')
 
-extra_package_file '/etc/dd-agent/conf.d/examples/<%= name %>.yaml.example'
+if linux?
+  conf_directory = "/etc/dd-agent/conf.d"
+elsif osx?
+  conf_directory = "/opt/datadog-agent/etc"
+elsif windows?
+  conf_directory = "../../extra_package_files/EXAMPLECONFSLOCATION"
+end
+
+extra_package_file "#{install_dir}/../../embedded/lib/python2.7/site-packages/"
+extra_package_file "#{conf_directory}/examples/"
```

Comment:
`extra_package_file` doesn't work on Windows (I don't think it fails the build though), see my previous review.